site/content/en/docs/tutorials/setup_minikube_in_china.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "minikube start"
+linkTitle: "Get Started!"
+weight: 1
+---
+
+This Tutorial explain how to setup minikube in China.
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">1</strong></span>Installation</h2>
+
+### Binary download
+
+For more detail please refer to [minikube installation guide]({{<ref "/docs/start">}})
+
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">2</strong></span>Start your cluster</h2>
+
+Open a terminal. If on Windows, open a terminal with administrator access. Run:
+
+```shell
+minikube start --image-repository=registry.cn-hangzhou.aliyuncs.com/google_containers
+```
+If minikube fails to start, see the [drivers page]({{<ref "/docs/drivers">}}) for help setting up a compatible container or virtual-machine manager.
+
+For users who use the Hyper-V environment, you should first open the Hyper-V manager to create an external virtual switch. After that, we can use the following commands to create a Hyper-V-based Kubernetes test environment:
+
+```shell
+.\minikube.exe start --image-mirror-country cn \
+    --iso-url=https://kubernetes.oss-cn-hangzhou.aliyuncs.com/minikube/iso/minikube-v1.12.0.iso \
+    --registry-mirror=https://xxxxxx.mirror.aliyuncs.com \
```

Comment:
Can we provide more info for this `xxxxxx` here? Thanks

Reply:
Agreed. https://github.com/kubernetes/minikube/issues/5020#issuecomment-739334195 has great instructions there. Do you mind integrating what @Hongbo-Miao has suggested there?

Reply:
sure

Reply:
updated as per @Hongbo-Miao suugestion.